ply-core/ply-core.cabal

```diff
@@ -5,6 +5,11 @@ author:             Chase <chase@mlabs.city>
 license:            MIT
 extra-source-files: ../README.md
 
+flag new-ledger-namespace
+  description: Use the new plutus-ledger-api namespace (0a8b1ad)
+  manual:      True
+  default:     False
```

Comment:
One final thing: I was thinking about the flag as you mentioned before - and I think the default should actually be `True`. After all, we _want_ people to use the new ledger namespace (and in the future, everyone certainly will). It also makes all of this forwards compatible.

People who _don't_ specify this flag (i.e their projects have already been updated) should never have to change their configs. So we can safely remove this flag in the future.